src/js/CreateAnalysis/Components/EnhancedProgressBar.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import {withStyles} from "@material-ui/core/styles";
+import Stepper from "@material-ui/core/Stepper";
+import Step from "@material-ui/core/Step";
+import StepLabel from "@material-ui/core/StepLabel";
+
+const styles = theme => ({
+  root: {
+    width: "100vw"
+  },
+  backButton: {
+    marginRight: theme.spacing.unit
+  },
+  instructions: {
+    marginTop: theme.spacing.unit,
+    marginBottom: theme.spacing.unit
+  }
+});
+
+function getSteps() {
```

Comment:
You could probably define this as a constant for now.

const STEPS = ["Add input files", "Specify metadata info", "Create dashboard"]